crates/sui-replay/src/displays/gas_status_displays.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::displays::Pretty;
+use std::fmt::{Display, Formatter};
+use sui_types::gas::SuiGasStatus;
+use sui_types::gas_model::gas_v2::SuiGasStatus as GasStatusV2;
+use tabled::{
+    builder::Builder as TableBuilder,
+    settings::{style::HorizontalLine, Style as TableStyle},
+};
+
+impl<'a> Display for Pretty<'a, SuiGasStatus> {
```

Comment:
Lets also add a convince method on SuiGasStatus for ease of converting to this pretty format. eg:

fn pretty(&self) -> Pretty<'a, Self> {
}

Then you can just call .pretty() easily in a format string

Reply:
I really like that look of calling `.pretty()`. As things are currently structured, the replay crate imports the crate with `SuiGasStatus`, and if we add that method on `SuiGasStatus` we would need to import `Pretty` from the replay crate. `Pretty` is to be used by other types as well for replay output, so I think it is better to keep it in the replay crate.

Reply:
Ah i didn't realize that this format lived only in the replay crate.